CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * New interactive command `clojure-cycle-not`.
 * New defcustom `clojure-comment-regexp` for font-locking `#_` or `#_` AND `(comment)` sexps.
 * [#459](https://github.com/clojure-emacs/clojure-mode/issues/459): Add font-locking for new built-ins added in Clojure 1.9.
+* [#471](https://github.com/clojure-emacs/clojure-mode/pull/471): Support tagged maps, new in Clojure 1.9
```

Comment:
I think the link is not correct - looks like a pull request link, not an issue link. Also, I guess you meant to say this adds support for Paredit, right? 

This sentence should end with a `.`, btw.

Reply:
It doesn't add support for paredit, it fixes the existing support.

Reply:
Yeah, yeah. I meant that what you wrote could be interpreted as the addition of some broader set of functionality for namespaced maps, so I wanted the entry to be more specific. 

Btw, @benedekfazekas, that's a nice idea for a simple refactoring - making a map namespaced/unnamespaced.